core/src/main/resources/hudson/logging/LogRecorder/configure.jelly

```diff
@@ -54,7 +54,12 @@ THE SOFTWARE.
                 ${%Logger}
               </th>
               <td width="60%">
-                <f:textbox field="name" autoCompleteUrl="autoCompleteLoggerName" />
+                <table width="100%">
```

Comment:
this change needs testing with: https://github.com/jenkinsci/jenkins/pull/3895

nested tables are known to cause issues, it's due for merging into this weekly so let's hold off on this till it's merged

Reply:
Will rebase and see how it looks. I guess I should work on converting to `div` as well for that component.